test/active_job/queue_adapters/adapter_testing/dispatch_jobs.rb

```diff
@@ -0,0 +1,15 @@
+module ActiveJob::QueueAdapters::AdapterTesting::DispatchJobs
+  extend ActiveSupport::Concern
```

Comment:
Oh, I know you copied this one from other similar modules to test the adapter, but I believe we don't need this, since we aren't using any of the `ActiveSupport::Concern` features.